File: server/test-data/rename/python/project1/rename_test.py
import file2
from file1 import *

test = test_class(0)
in1 = test_class.inner1()
in2 = test_class.inner1.inner2()

file2.test = file2.test_class(0)
file2.in1 = file2.test_class.inner1()
file2.in2 = file2.test_class.inner1.inner2()

curr = test.cur + 1
test.foo()

File: server/test-data/rename/java/project1/src/RenameTest.java
import org.TestClass;

public class RenameTest {
  public RenameTest() {
    TestClass test = new TestClass(0);
    TestClass.Inner1 in1 = new TestClass.Inner1();
    TestClass.Inner1.Inner2 in2 = new TestClass.Inner1.Inner2();

    com.TestClass comTest = new com.TestClass(0);
    com.TestClass.Inner1 comIn1 = new com.TestClass.Inner1();
    com.TestClass.Inner1.Inner2 comIn2 = new com.TestClass.Inner1.Inner2();

    int bbb = test.aaa + 1;
    test.foo();
  }
}